diff --git a/src/decoder/plugins/FfmpegDecoderPlugin.cxx b/src/decoder/plugins/FfmpegDecoderPlugin.cxx
index 0f8abdd..60232d7 100644
--- a/src/decoder/plugins/FfmpegDecoderPlugin.cxx
+++ b/src/decoder/plugins/FfmpegDecoderPlugin.cxx
@@ -776,6 +776,7 @@ static const char *const ffmpeg_mime_types[] = {
 	"audio/qcelp",
 	"audio/vorbis",
 	"audio/vorbis+ogg",
+	"audo/wav",
 	"audio/x-8svx",
 	"audio/x-16sv",
 	"audio/x-aac",
